src/cpu/test_lockit/coma.cc

```diff
@@ -274,25 +274,16 @@ int COMAI_HI::aite_hyouka(const TLColor ba3[6][kHeight], TLColor tsumo[])
 int COMAI_HI::hyouka(const TLColor ba3[6][kHeight], TLColor tsumo[], int zenkesi_own, TLColor aite_ba[6][kHeight], int zenkesi_aite)
 {
     TLColor ba[6][kHeight] {};
-    TLColor ba_a[6][kHeight] {};
     TLColor ba2[6][kHeight] {};
-    int point[6][12];
-    int i, j;
-    int num = 0;
-    int n;
+    TLColor ba_ee[6][kHeight];
+    TLColor ba_a[6][kHeight] {};
```

Comment:
Can't we move some of these to just before calling copyField() ?
Not sure, though.